moodlehq/moodlemobile2
# Mobile-1713 choice: Changed choice results to use chart.js

Swapped the choice results in the app to using chart.js.
Unsure how to deal with the strings that are not being used anymore, some help on this would be appreciated.
Test:
1. Create a choice activity, allow the choices to be published.
2. Log in as students and vote on the choice activity. 
3. Log in as person who can see the results.
